### PR TITLE
bulk change `type T` to `type *T*` in spec

### PR DESCRIPTION
**Pull request by pelszkow**
### Details:
 - *bulk change `type T` to `type *T*` in spec*

### Tickets:
 - *59113*

### CODE REVIEW

File: docs/ops/movement/Tile_1.md [hunk 1]
```diff
@@ -14,12 +14,12 @@ No attributes available.
 
 **Inputs**:
 
-* **1**: "data" - an input tensor to be padded. A tensor of type T1. **Required.**
-* **2**: "repeats" - a per-dimension replication factor. For example, *repeats* equal to 88 means that the output tensor gets 88 copies of data from the specified axis. A tensor of type T2. **Required.**
+* **1**: "data" - an input tensor to be padded. A tensor of type *T*1. **Required.**
```
Comment (ggalieroc) [inline docs/ops/movement/Tile_1.md:17]: \*T\*1 ---> \*T1\*

File: docs/ops/movement/Tile_1.md [hunk 2]
```diff
@@ -14,12 +14,12 @@ No attributes available.
...
+* **2**: "repeats" - a per-dimension replication factor. For example, *repeats* equal to 88 means that the output tensor gets 88 copies of data from the specified axis. A tensor of type *T*2. **Required.**
```
Comment (ggalieroc) [inline docs/ops/movement/Tile_1.md:18]: \*T\*2 ---> \*T2\*

File: docs/ops/sequence/CTCGreedyDecoderSeqLen_6.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ The main difference between [CTCGreedyDecoder](CTCGreedyDecoder_1.md) and CTCGre
 
 * **1**: `data` - input tensor of type *T_F* of shape `[N, T, C]` with a batch of sequences. Where `T` is the maximum sequence length, `N` is the batch size and `C` is the number of classes. **Required.**
 
-* **2**: `sequence_length` - input tensor of type *T_I* of shape `[N]` with sequence lengths. The values of sequence length must be less or equal to `T`. **Required.**
+* **2**: `sequence_length` - input tensor of type *T_I* of shape `[N]` with sequence lengths. The values of sequence length must be less or equal to *T*. **Required.**
```
Comment (ggalieroc) [inline docs/ops/sequence/CTCGreedyDecoderSeqLen_6.md]: In this case T is dimension, it should be \`T\`

File: docs/ops/movement/Tile_1.md (hunk 1 above)
Comment (pelszkow) [reply docs/ops/movement/Tile_1.md]: done

File: docs/ops/movement/Tile_1.md (hunk 2 above)
Comment (pelszkow) [reply docs/ops/movement/Tile_1.md]: done

File: docs/ops/sequence/CTCGreedyDecoderSeqLen_6.md (hunk 3 above)
Comment (pelszkow) [reply docs/ops/sequence/CTCGreedyDecoderSeqLen_6.md]: done

**Review by ggalieroc — APPROVED** [review]

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**